frontend/src/app/main/data/workspace/libraries.cljs

```diff
@@ -529,14 +529,15 @@
         (st/emit! (rt/nav-new-window :workspace pparams qparams))))))
 
 (defn ext-library-changed
-  [file-id modified-at changes]
+  [file-id modified-at revn changes]
   (us/assert ::us/uuid file-id)
   (us/assert ::cp/changes changes)
   (ptk/reify ::ext-library-changed
     ptk/UpdateEvent
     (update [_ state]
       (-> state
-          (assoc-in [:workspace-libraries file-id :modified-at] modified-at)
+          (update-in [:workspace-libraries file-id]
+                     #(assoc % :modified-at modified-at :revn revn))
```

Comment:
you don't need to ceate the `#(assoc ...)` anon function:

```clojure
(update-in [:workspace-libraries file-id] assoc :modified-at modified-at :revn revn)
```

Reply:
done